src/test/kotlin/com/anaplan/engineering/vdmprettyprinter/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.kt]
+trim_trailing_whitespace = false
```

Comment:
Looks like the pretty printing of the record data type does not properly trim spaces after `::`. This is more of a work-around, since I couldn't easily find the right place to fix that. Also, I didn't want to make any functional changes in this PR.